providers/io/osf/normalizer.py

```diff
@@ -66,7 +66,7 @@ class Project(Parser):
         ctx.embeds.affiliated_institutions.data
     )
     date_updated = tools.ParseDate(ctx.attributes.date_modified)
-    subject = tools.Delegate(Tag, ctx.attributes.category)
+    subject = tools.Delegate(Tag, tools.Maybe(ctx, 'attributes.category'))
```

Comment:
Oooh yeah, good call!

...but it doesn't seem like Map filters out falsey values. I deleted subject and made tags:
```
tags = tools.Map(tools.Delegate(ThroughTags), ctx.attributes.category, ctx.attributes.tags)
```
but that first tag still has an empty name for uncategorized projects. 😢